packages/oas/medusa-oas-cli/.npmignore

```diff
@@ -0,0 +1,10 @@
+src
```

Comment:
todo: rm in favor or the package.json files: ["dist"]

Reply:
Ah yes, I much prefer that approach. Done in b05953ed70696f10cb783522fde39362906578d6.